repository/gorm/soundboard.go

```diff
@@ -0,0 +1,39 @@
+package gorm
+
+import (
+	"github.com/gofrs/uuid"
+	"github.com/traPtitech/traQ/model"
+)
+
+func (repo *Repository) CreateSoundboardItem(soundID uuid.UUID, soundName string, stampID *uuid.UUID, creatorID uuid.UUID) error {
+	return repo.db.Create(&model.SoundboardItem{
+		ID:        soundID,
+		Name:      soundName,
+		StampID:   stampID,
+		CreatorID: creatorID,
+	}).Error
+}
+
+func (repo *Repository) GetAllSoundboardItems() ([]*model.SoundboardItem, error) {
+	items := make([]*model.SoundboardItem, 0)
+	if err := repo.db.Find(&items).Error; err != nil {
+		return nil, err
+	}
+	return items, nil
+}
+
+func (repo *Repository) GetSoundboardByCreatorID(creatorID uuid.UUID) ([]*model.SoundboardItem, error) {
+	items := make([]*model.SoundboardItem, 0)
+	if err := repo.db.Where("creator_id = ?", creatorID).Find(&items).Error; err != nil {
```

Comment:
非常に細かいところなので無視していただいても構わないんですが、単純な条件であれば自動で条件マッピングをしてくれるこちらの記法のほうがいいかもしれません…！
```suggestion
	if err := repo.db.Where(&model.SoundboardItem{CreatorID: creatorID}).Find(&items).Error; err != nil {
```